FAIL
Kernel panic in run: PANIC at ../../threads/synch.c:234 in lock_release(): assertion `lock_held_by_current_thread (lock)' failed.
Call stack: 0xc0107518 0xc0102996 0xc010a2bc 0xc0100cb1
Translation of call stack:
0xc0107518: debug_panic (.../../lib/kernel/debug.c:34)
0xc0102996: lock_release (...../../threads/synch.c:236)
0xc010a2bc: b_thread_func (...rity-donate-multiple.c:76)
0xc0100cb1: kernel_thread (..../../threads/thread.c:412)
